hw/top_earlgrey/top_earlgrey.core

```diff
@@ -73,3 +84,13 @@ targets:
     parameters:
       - PRIM_DEFAULT_IMPL=prim_pkg::ImplGeneric
     toplevel: top_earlgrey
+  asic-synth:
+    default_tool: icarus
```

Comment:
I've filed https://github.com/olofk/edalize/issues/89 to get DC support added to edalize, please add a TODO comment with this issue number/link.

Reply:
noted, thanks. I'll file an issue for JasperGold as well.